src/main/resources/data/tconstruct/scripts/tconstruct/tinker_station.zs

```diff
@@ -0,0 +1,83 @@
+<recipetype:tconstruct:tinker_station>.addUpgradeModifierRecipe("upgrade_modifier_example", [<item:minecraft:red_sand>], <tag:items:tconstruct:modifiable>, "tconstruct:tasty", 1, 3, 1);
+<recipetype:tconstruct:tinker_station>.addUpgradeModifierRecipe("upgrade_modifier_requirements_example", [<item:minecraft:white_wool>], <tag:items:tconstruct:modifiable>, "tconstruct:reach", 1, 3, 2, {"tconstruct:haste": 1}, 1, "recipe.tconstruct.modifier.requirements_error");
+<recipetype:tconstruct:tinker_station>.addAbilityModifierRecipe("ability_modifier_example", [<item:minecraft:gravel>], <tag:items:tconstruct:modifiable>, "tconstruct:smite", 1, 3, 1);
+<recipetype:tconstruct:tinker_station>.addAbilityModifierRecipe("ability_modifier_requirements_example", [<item:minecraft:orange_wool>], <tag:items:tconstruct:modifiable>, "tconstruct:sharpness", 1, 3, 1, {"tconstruct:emerald": 1}, 1, "recipe.tconstruct.modifier.requirements_error");
+<recipetype:tconstruct:tinker_station>.addIncrementalModifierRecipe("incremental_modifier_example", <item:minecraft:black_wool>, 2, 8, <tag:items:tconstruct:modifiable>, "tconstruct:haste", 1, 8, <item:minecraft:black_dye>);
+<recipetype:tconstruct:tinker_station>.addIncrementalModifierRecipe("incremental_modifier_requirements_example", <item:minecraft:yellow_wool>, 3, 8, <tag:items:tconstruct:modifiable>, "tconstruct:sharpness", 1, 8, <item:minecraft:yellow_dye>, {"tconstruct:emerald": 1}, 1, "recipe.tconstruct.modifier.requirements_error");
+<recipetype:tconstruct:tinker_station>.addIncrementalUpgradeModifierRecipe("incremental_upgrade_modifier_example", <item:minecraft:blue_wool>, 4, 20, <tag:items:tconstruct:modifiable>, "tconstruct:bane_of_arthropods", 1, 3, 1, <item:minecraft:blue_dye>);
+<recipetype:tconstruct:tinker_station>.addIncrementalUpgradeModifierRecipe("incremental_upgrade_modifier_requirements_example", <item:minecraft:red_wool>, 4, 20, <tag:items:tconstruct:modifiable>, "tconstruct:fiery", 1, 3, 1, <item:minecraft:red_dye>, {"tconstruct:diamond": 1}, 1, "recipe.tconstruct.modifier.requirements_error");
+<recipetype:tconstruct:tinker_station>.addIncrementalAbilityModifierRecipe("incremental_ability_modifier_example", <item:minecraft:brown_wool>, 1, 2, <tag:items:tconstruct:modifiable>, "tconstruct:luck", 1, 4, 1, <item:minecraft:brown_dye>);
+<recipetype:tconstruct:tinker_station>.addIncrementalAbilityModifierRecipe("incremental_ability_modifier_requirements_example", <item:minecraft:magenta_wool>, 1, 2, <tag:items:tconstruct:modifiable>, "tconstruct:antiaquatic", 1, 4, 1, <item:minecraft:magenta_wool>, {"tconstruct:diamond": 1, "tconstruct:haste": 2}, 2, "recipe.tconstruct.modifier.requirements_error");
+
+/*
+ * Removes the Expanded upgrade from the Tinker Station.
+ */
+
+// <recipetype:tconstruct:tinker_station>.removeByName(name as string)
+
+<recipetype:tconstruct:tinker_station>.removeByName("tconstruct:tools/modifiers/upgrade/expanded");
```

Comment:
That recipe was wrongly put in the upgrade folder, should have been in ability, so that path no longer exists.

You want `"tconstruct:tools/modifiers/ability/expanded"` probably